src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -13170,3 +13170,103 @@ def test_aks_disable_ssh(self, resource_group, resource_group_location):
 
         # delete
         self.cmd('aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+    @live_only()
```

Comment:
May I ask why this case is marked as `@live_only()`? If there is a special reason, please leave a comment. Or remove this. BTW, if you really want to mark the case as `@live_only()`, you should remove the useless recording file `test_aks_create_with_pod_ip_allocation_mode_static_block.yaml`

Reply:
Removed the @liveonly tag. I used that when I was trying to fix the tests and used it from another testcase in the file as an example. Updated now to remove that tag